.prettierrc.cjs

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	useTabs: true,
```

Comment:
let's do spaces, `tabWidth: 2` and `semi: false`

Reply:
Updated to fix; tabWidth defaults to 2 in prettier, so left that be, but can explicitly define if you'd prefer!